srcpkgs/reflex/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'reflex'
+pkgname=reflex
+version=20210808
+revision=1
+_date=${version##*.}
+wrksrc="${pkgname}-${_date}"
+build_style=gnu-configure
+makedepends="byacc"
```

Comment:
Shouldn't this be in hostmakedepends?

Reply:
I changed makedpends for hostMakeedpends, but the failure problems are "Connection timed out" and "Operation timed out" occurs when you try to download the tgz package, aarch64-linux-musl compiled normally without fail.

Reply:
Looking at all compilation checks, you do not have errors when compiling for no architecture, failures are only when you try to download the tgz package, sometimes normal download and compile, others can not download.

Reply:
Resolved, removed a line, now passed on all architectures!